src/sdltiles.cpp

```diff
@@ -22,6 +22,7 @@
 #include <type_traits>
 #include <tuple>
 
+#include "platform_win.h"
```

Comment:
If it is not used on non-Windows, it should be guarded.

Reply:
`platform_win.h` itself has windows-related guards, and should be safe to include anywhere. This is why I opted to put it here, rather than inside the guards below.

But if you feel strongly about this, I'm happy to move it.

Reply:
I afraid it could be missing on a non-Windows system.